lib/fluent/plugin/formatter_syslog_rfc5424.rb

```diff
@@ -26,15 +26,14 @@ def configure(conf)
       def format(tag, time, record)
         log.debug("Record")
         log.debug(record.map { |k, v| "#{k}=#{v}" }.join('&'))
-
```

Comment:
Nit: can we revert the empty line so that two blocks inside function will be more readable

Reply:
Acknowledged